taichi/platform/opengl/opengl_api.cpp

```diff
@@ -286,6 +286,12 @@ bool is_opengl_api_available() {
   return true;
 }
 
+int opengl_get_threads_per_group() {
+  int ret = 1792;
```

Comment:
Is this magic number (`1792`) here necessary?

Reply:
Better that a random initial state :) Maybe I should change that to `1`.